src/presentation/onboarding/seed-confirm/index.tsx

```diff
@@ -6,7 +6,7 @@ import { INITIALIZE_END_OF_FLOW_ROUTE } from '../../routes/constants';
 import Shell from '../../components/shell';
 import { useDispatch } from 'react-redux';
 import { ProxyStoreDispatch } from '../../../application/redux/proxyStore';
-import { setVerified } from '../../../application/redux/actions/wallet';
+import { setOnboardingVerified } from '../../../application/redux/actions/onboarding';
 
 const NULL_ERROR = '';
 const ERROR_MSG = 'Invalid mnemonic';
```

Comment:
The string 'Invalid mnemonic' is also being used here:
https://github.com/louisinger/marina/blob/account/src/domain/mnemonic.ts#L9

Should we put it in the constants file?